qfieldsync/core/offline_converter.py

```diff
@@ -47,8 +47,9 @@ class OfflineConverter(QObject):
     task_progress_updated = pyqtSignal(int, int)
     total_progress_updated = pyqtSignal(int, int, str)
 
-    def __init__(self, project, export_folder, extent, offline_editing):
+    def __init__(self, iface, project, export_folder, extent, offline_editing):
```

Comment:
I always try to keep iface out as much as possible. Then it's also available to standalone apps.

Reply:
e.g. here there is already project, can we just call clear on it?

Reply:
that would work in master, but in 2.18 QgsProject::clear() is not doing a lot of stuff (I believe it's because composers are not attached to the project).

Reply:
Ok, let's go with this for now. Might be too much to create a callback just for that. Let's deal with this if we actually run into issues.